homeassistant/components/pseg/sensor.py

```diff
@@ -0,0 +1,150 @@
+"""Platform for sensor integration."""
+from datetime import timedelta
+import logging
+
+import voluptuous as vol
+
+from pseg import Meter
+from pseg import MeterError
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_ENERGIZE_ID = "energize_id"
+CONF_ACCESS_TOKEN = "session_id"
```

Comment:
```suggestion
CONF_SESSION_ID = "session_id"
```

or 

```suggestion
CONF_ACCESS_TOKEN = "access_token"
```

Reply:
We can import the access token constant from const.py:
https://github.com/home-assistant/home-assistant/blob/de1799d486bf0c55c14aa8672f6e0c84d454443b/homeassistant/const.py#L27